internal/cmd/agent/root.go

```diff
@@ -33,7 +33,8 @@ type FleetAgent struct {
 	command.DebugConfig
 	AgentScope string `usage:"An identifier used to scope the agent bundleID names, typically the same as namespace" env:"AGENT_SCOPE"`
 	UpstreamOptions
-	CheckinInterval string `usage:"How often to post cluster status" env:"CHECKIN_INTERVAL"`
+	CheckinInterval      string `usage:"How often to post cluster status" env:"CHECKIN_INTERVAL"`
+	EnableLeaderElection bool   `name:"leader-elect" default:"true" usage:"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager."`
```

Comment:
```suggestion
	EnableLeaderElection bool   `name:"leader-elect" default:"true" usage:"Enable leader election for the agent. Enabling this will ensure there is only one active agent."`
```